README.md

```diff
@@ -494,6 +495,43 @@ jobs:
           password: ${{ secrets.DIGITALOCEAN_ACCESS_TOKEN }}
 ```
 
+### Buildkite Package Registry
+
+Use your Buildkite registered email address to generate an API access token to authenticate. U
+
+```yaml
+name: ci
+
+on:
+  push:
+    branches: main
+
+jobs:
+  login:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Login to Buildkite Package Registry
+        uses: docker/login-action@v3
+        with:
+          registry: packages.buildkite.com
+          username: buildkite #All usernames to buildkite packages is "buildkite". 
```

Comment:
Don't inline comment also doesn't look necessary as this is documented in buildkite docs already: https://buildkite.com/docs/package-registries/container#publish-an-image